src/components/NavBar.js

```diff
@@ -17,18 +21,23 @@ function NavBar() {
         </Link>
 
       <div>
-        <button
-          type="button"
-          onClick={() => history.push('/signin')}
-        >
-          Log in
-        </button>
-        <button
-          type="button"
-          onClick={() => history.push('/signup')}
-        >
-          Registreren
-        </button>
+        {isAuth && (
+            <button type="button" onClick={logout}>
+              Logout
+            </button>
+        )}
+
+        {!isAuth && (
+            <button type="button" onClick={() => navigate('/signin')}>
+              Login
+            </button>
+        )}
+
+        {!isAuth && (
+            <button type="button" onClick={() => navigate('/signup')}>
+              Registeren
+            </button>
+        )}
```

Comment:
In principe prima zo, maar als er nog meer bij zou komen dan zie ik liever:
```suggestion
        {isAuth ? (
            <button type="button" onClick={logout}>
              Logout
            </button>
        ) : (
            <>
                <button type="button" onClick={() => navigate('/signin')}>
                    Login
                </button>
                <button type="button" onClick={() => navigate('/signup')}>
                    Registeren
                </button>
            </>
        )}
```